src/app.service.js

```diff
@@ -226,4 +232,34 @@ export class AppService {
   formatHttpURL(url) {
     return url.replace(/^(?!https?:\/\/)/, "https://");
   }
+
+  getVelogMainContent(bodyElement) {
+    const allElements = Array.from(bodyElement.querySelectorAll("*"));
+
+    const titleElement = allElements.find((element) => {
+      return (
+        typeof element.className === "string" &&
+        element.className.toLowerCase().includes("tbwpx")
+      );
+    });
+
+    const mainElements = allElements.filter((element) => {
+      return (
+        typeof element.className === "string" &&
+        element.className.toLowerCase().includes("dftzxp")
```

Comment:
```suggestion
        element.className.toLowerCase().includes(VELOG_MAIN_CLASS_NAME)
```

`const VELOG_MAIN_CLASS_NAME = "dftzxp";`

Reply:
해당 사항 반영하였습니다.